src/libtrx/include/libtrx/game/anims/util.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define CMD_ENVIRONMENT_BITS(C) ((C & 0xC000) >> 14)
+#define CMD_PARAM_BITS(C) (C & 0x3FFF)
```

Comment:
Since it's used in the client code let's prefix it with ANIM_. Not very convinced in my proposal though.